examples/tutorials/hello-gltf/app.ts

```diff
@@ -59,14 +59,16 @@ export default class AppAnimationLoopTemplate extends AnimationLoopTemplate {
       const u_MVPMatrix = new Matrix4(projectionMatrix)
         .multiplyRight(viewMatrix)
         .multiplyRight(worldMatrix);
-      model.setUniforms({
```

Comment:
I'm not sure why [this error](https://github.com/visgl/luma.gl/actions/runs/10317558240/job/28562100092?pr=2173) is coming up, it doesn't happen for me locally